.env

```diff
@@ -133,4 +133,6 @@ EXPOSE_API=true
 
 ENABLE_ASSISTANTS=false #set to true to enable assistants feature
 
-ALTERNATIVE_REDIRECT_URLS=`[]` #valide alternative redirect URL for OAuth
\ No newline at end of file
+ALTERNATIVE_REDIRECT_URLS=`[]` #valide alternative redirect URL for OAuth
+
+SLACK_URL_REPORT_ASSISTANT=#provide https://api.slack.com/messaging/webhooks to get notified when an assistant gets reported
```

Comment:
Is this a secret or will it be in the public  `.env.template` ?

Reply:
First you'll need to add it [here](https://github.com/huggingface/chat-ui/settings/secrets/actions) on github

Then you'll need to update [the script that generates the prod config file](https://github.com/huggingface/chat-ui/blob/main/scripts/updateProdEnv.ts) should be fairly straight forward, just get `process.env.WEBHOOK_URL_REPORT_ASSISTANT` and add it to `full_config` 


Then optionally for local testing you can add `WEBHOOK_URL_REPORT_ASSISTANT=<here>` to your `.env.SECRET_CONFIG` but it's not a requirement. (only useful if you make use of `npm run updateLocalEnv` )

Reply:
> First you'll need to add it [here](https://github.com/huggingface/chat-ui/settings/secrets/actions) on github

done

> Then you'll need to update [the script that generates the prod config file](https://github.com/huggingface/chat-ui/blob/main/scripts/updateProdEnv.ts) should be fairly straight forward, just get process.env.WEBHOOK_URL_REPORT_ASSISTANT and add it to full_config

done 729789c1a5b213b1df9f8054551b20534dcdc35f